opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -107,6 +108,37 @@ class StackdriverExporter {
   StackdriverExporter() = delete;
 };
 
+class StackdriverExporterHandler
```

Comment:
Our intent for this API was that users only need to call `StackdriverExporter::Register(std::move(opts));`

How does subclassing the handler change this? Should we instead have a callback in the options?

Reply:
Okay, I added a std::function to the options class. Makes it a much simpler PR overall, only a few lines change. :). This works for me.

Reply:
Thanks!